components/Home/Home.jsx

```diff
@@ -102,7 +103,7 @@ export default () => {
         <CurrentCycle
           name={cycleDetails && cycleDetails[cycles.selectedCycleIndex].name}
           subtext={cycleDetails && cycleDetails[cycles.selectedCycleIndex].muscleGroups}
-          color={cycleDetails && COLORS[cycles.selectedCycleIndex % COLORS.length]}
+          color={cycleDetails && COLORS[colorTheme][cycles.selectedCycleIndex % (COLORS[colorTheme].length - 1)]}
```

Comment:
I think the < and > for the selected cycle are part of the purple color scheme, might be a good idea to change that color too because it feels a little out of place